src/codemagic/tools/app_store_connect.py

```diff
@@ -292,7 +292,7 @@ def get_latest_build_number(
         platform: Optional[Platform] = None,
     ) -> Optional[str]:
         """
-        Get the highest build number used for the given app considering both TestFlight and App Store submissions
+        Get the highest build number of the highest version used for the given app considering both TestFlight and App Store submissions
```

Comment:
```suggestion
        Get the highest build number of the highest version used for the
        given app considering both TestFlight and App Store submissions
```

Tests fail because of too long line.

Reply:
Please check if you like it. I deleted  the considering both TestFlight and App Store submission part